content/20.treatments.md

```diff
@@ -188,13 +188,121 @@ Summarize the state of the antiviral approach.
+Another potential mechanism by which n-3 PUFA could exert beneficial effects against viral infections is by acting as precursor molecules for the biosynthesis of endogenous specialised proresolving mediators (SPM) like protectins and resolvins that actively resolve inflammation and infection [@doi:10.1016/j.immuni.2014.02.009]. 
+
+###### Current Evidence
+
+SPM have exhibited beneficial effects against a variety of lung infections including RNA viruses [@doi:10.1038/nri.2015.4].
+Indeed, protectin D1 has been shown to increase survival from H1N1 viral infection in mice by affecting the viral replication machinery [@doi:10.1016/j.cell.2013.02.027].
+Moreover, not all studies are in agreement that n-3 PUFA are effective against infections [@doi:10.3945/jn.109.108027]. 
+Indeed, the effectiveness of n-3 PUFA against infections is dependent on the dosage, timing, and the specific pathogens responsible [@doi:10.1016/j.jinf.2016.10.001].  
+
+###### Summary
+
+However, the overall lack of human studies in this area means there is limited evidence as to whether these nutraceuticals could affect COVID-19 infection.  
+
+##### Zinc Supplements
+
+There is evidence that nutrient supplements may exhibit some benefit against RNA viral infections.
+Zinc is a trace metal obtained from dietary sources or supplementation that is important for the maintenance of immune cells involved in adaptive and innate immunity [@doi:10.1016/j.abb.2016.03.022].
+Zinc supplements can be administered orally as a tablet or as a lozenge and they are available in many forms such zinc picolinate, zinc acetate, and zinc citrate.
+Zinc is also available from dietary sources including meat, seafood, nuts, seeds, legumes, and dairy.
+
+###### Potential Mechanisms
+
+The role of zinc in immune function has been extensively reviewed [@doi:10.1016/j.abb.2016.03.022].
+Zinc is an important signaling molecule and zinc levels can alter host defense systems. 
+In inflammatory situations such as an infection, zinc can regulate leukocyte immune responses and it can activate the nuclear factor kappa-light-chain-enhancer of activated B cells (NF-kB), thus altering cytokine production [@doi:10.4049/jimmunol.179.6.4180; @doi:10.1067/mlc.2001.118108]. 
+In particular, zinc supplementation can increase natural killer cell levels, which are important cells for host defense against viral infections [@doi:10.1126/science.1198687; @doi:10.1016/j.abb.2016.03.022].
+
+###### Current Evidence
+
+Adequate zinc intake has been associated with reduced incidence of infection [@doi:10.1093/ajcn/85.3.837] and antiviral immunity [@doi:10.1093/advances/nmz013].
+Conversely, zinc deficiency is associated with increased susceptibility to infection in the elderly, which can be prevented when zinc supplements are administered [@doi:10.1093/ajcn/85.3.837].
```

Comment:
```suggestion
Similarly, a randomized, double-blind, placebo-controlled trial that administered zinc supplementation to elderly subjects over the course of a year found zinc deficiency to be associated with increased susceptibility to infection and that zinc deficiency could be prevented through supplementation [@doi:10.1093/ajcn/85.3.837].
```
Apologies, I have a few more questions!
1. Is "conversely" this right word? The framing is swapped between the two sentences, but the ideas don't seem at odds -- I'm reading this as both the general population and the elderly in particular need to avoid zinc deficiency. 
2. I checked out that paper because I was curious. It seems pretty well designed to my eye? Again, I've been a genomics person for the last ~8 years, it's been a while since I thought about anything double-blind or placebo-controlled! But I fleshed out the description a little, don't feel like you have to accept!
3. As someone who knows almost nothing about neutraceuticals, I know being vitamin deficient is bad for your health (e.g., scurvy and vitamin C), but what I am unsure of is what kinds of supplementation can help -- namely, do modern vitamin C supplements do anything or should we all just be trying to eat limes? If this study showed that non-diet zinc supplementation actually helps prevent zinc deficiency, and also that zinc deficiency can affect disease risk, I personally think that's an interesting detail!
3. Looking at this paper, it actually seems quite closely tied in with some of the other sections that have touched on cytokines (for pharmaceuticals). We might want to keep an eye on flow between these two sections since they seem closely related!